rental.rb

```diff
@@ -0,0 +1,17 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    book.rentals_list.push(self)
+    person.rentals_list.push(self)
+  end
+
+  def add_book(book)
+    @book = book
+    book.rentals_list.push(self) unless book.rentals_list.include?(self)
+  end
```

Comment:
- [ ] [OPTIONAL] As already suggested by the previous reviewer, please remove the `add_book` method if it is not in use.